script/verify-changes.test.ts

```diff
@@ -0,0 +1,10 @@
+import { execaCommand } from "execa";
+import { expect } from "vitest";
+import { test } from "vitest";
+
+import { filesExpectedToBeChanged } from "./constants.js";
+
+test.each([...filesExpectedToBeChanged])("verify %s", async (file) => {
+	const { stdout } = await execaCommand(`git diff HEAD -- ${file}`);
+	expect(stdout).toMatchSnapshot();
```

Comment:
We can try ignoring these files, or just diff part of those files. The first one would stop verifying which I don't think is the ideal solution, and the second solution might be complicated (AFAIK), which one do you prefer?

Reply:
I'd prefer the latter for completeness yeah. The code could apply some kind of filter to them or instead of using snapshots use like a [`.toMatch`](https://vitest.dev/api/expect#tomatch). But if that's too much for this PR I think it'd be fine to ignore them for now and file a followup issue.